### PR TITLE
chore(logging): Adds logrus setup back and changes klog verbosity levels

### PR DESCRIPTION
**Pull request by jpower432**
Signed-off-by: Jennifer Power <barnabei.jennifer@gmail.com>

# Description

This PR does some normalization after the migration from `logrus` to `klog` as the main logging solution.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [X] Unit tests updated
- [X] Manual testing

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pkg/cli/mirror/create.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func (o *MirrorOptions) run(ctx context.Context, cfg *v1alpha2.ImageSetConfigura
 	}
 
 	if len(cfg.Mirror.Samples) != 0 {
-		klog.V(4).Info("sample images full not implemented")
+		klog.Info("sample images full not implemented")
```
Comment (dinhxuanvu) [inline pkg/cli/mirror/create.go:147]: Not sure why we move away from verbosity setting for this logging? Is this intentional?

**Review by dinhxuanvu — COMMENTED** [review]
I think the changes look good. Honestly, we have no criteria to decide which log is in which level is appropriate. So this is kind of intuitive assessment. I don't have a problem with it but it is something to acknowledge.

**Comment by jpower432** [comment]
> I think the changes look good. Honestly, we have no criteria to decide which log is in which level is appropriate. So this is kind of intuitive assessment. I don't have a problem with it but it is something to acknowledge.

That's fair. I tried to match it up best I could with `oc` to have a consistent log output.

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Pull request merged**